frontend/src/features/admin-form/settings/components/WorkflowSettingsSection/WorkflowSettingsSection.tsx

```diff
@@ -0,0 +1,29 @@
+import { Text } from '@chakra-ui/react'
+
+import { FormResponseMode } from '~shared/types'
+
+import { GUIDE_TWILIO } from '~constants/links'
+import Link from '~components/Link'
+
+import { useAdminFormSettings } from '../../queries'
+
+import { WorkflowDetailsInput } from './WorkflowDetailsInput'
+
+export const WorkflowSettingsSection = (): JSX.Element => {
+  const { data: settings, isLoading } = useAdminFormSettings()
+  return settings?.responseMode === FormResponseMode.Multirespondent ? (
+    <>
+      <Text mb="2.5rem">
+        Create a workflow to collect responses from multiple respondents. We
+        currently support up to three respondents.&nbsp;
+        <Link href={GUIDE_TWILIO} isExternal>
+          Learn more about setting up a workflow
+        </Link>
+      </Text>
+
+      <WorkflowDetailsInput settings={settings} />
+    </>
+  ) : (
+    <></>
```

Comment:
prefer this to be extracted as guard statements above return / render

Reply:
deleted this component [245d990](https://github.com/opengovsg/FormSG/pull/6968/commits/245d99044c159a86406da30f9c6fcf11dfc11230)